package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/ccd-case-ui-toolkit",
-  "version": "3.1.2",
+  "version": "3.1.2-alpha-pagination-1",
```

Comment:
version needs to be upgraded with LIVE version

Reply:
@rayliangatos  changed it full version now.